docs/index.rst

```diff
@@ -180,6 +180,10 @@ ar71xx-generic
   - My Net N600
   - My Net N750
 
+* 8devices
+
+  - Carambola 2
```

Comment:
I'd rather put 8devices at the beginning of the list, numbers are usually sorted before alphabet chars.